bookkeeper-server/src/main/proto/BookkeeperProtocol.proto

```diff
@@ -85,13 +85,18 @@ message Request {
 message ReadRequest {
     enum Flag {
         FENCE_LEDGER = 1;
+        ENTRY_PIGGYBACK = 2;
     }
     optional Flag flag = 100;
     required int64 ledgerId = 1;
     // entryId will be -1 for reading the LAST_ADD_CONFIRMED entry.
     required int64 entryId = 2;
     // Used while fencing a ledger.
     optional bytes masterKey = 3;
+    // Used for waiting on last add confirmed update
+    optional int64 previousLAC = 4;
+    // Used as a timeout for the long polling request
```

Comment:
Please add time unit for the timeout. Just a mention in the comment would be good